packages/node_modules/pouchdb-adapter-websql-core/src/index.js

```diff
@@ -697,59 +701,58 @@ function WebSqlPouch(opts, callback) {
     }
 
     db.readTransaction(function (tx) {
+      // count the docs in parallel to other operations
+      countDocs(tx, function (docCount) {
+        totalRows = docCount;
+      });
 
```

Comment:
Incidentally in the previous version we were stupidly waiting for the `countDocs()` to callback before running the other queries. But since the result of `countDocs()` is only needed when the transaction was complete, this was a waste. We can just do it in parallel.